Fix/check memory (#49)
openweathermap/deker: docs/deker/fine_tuning.rst

   If you set a memory limit to your container, you'd better limit DEKER™ to the same value.
   Otherwise your container may be killed because of memory overflow.

``skip_collection_create_memory_check``
---------------------------------------
Currently deker has 3 places, where memory check, described in `memory_limit`_:

 * On collection creation via ``client.create_collection()``
 * On getting subset e.g ``array[:]``
 * On reading array as ``XArray`` e.g ``array[:].read_xarray()``

> While, last two prevents memory overflow and are required,
> sometimes you may need to be able to skip
> first one (e.g when you work with cluster)
> 
> You can do so, by providing

Reviewer: ```suggestion
While the last two prevent memory overflow and are required,
sometimes you may need to be able to skip the first one

You can do so by providing
```
